build.sbt

```diff
@@ -1,15 +1,17 @@
 soteriaAddSemantic := false
 
+scalaVersion in Global := "2.12.11"
```

Comment:
Remove this. For SBT plugin, the scalaVersion will be the one corresponding to the one provided by the sbt version

Reply:
The problem is that for some stupid reason, the default version seems to be 2.12.7, so if I don't have this, I get:
```[error] java.lang.RuntimeException: [SoteriaPlugin] [SoteriaPlugin] Scala: 2.12.7 != [2.12.11 OR 2.11.12] !!!
```

Reply:
There is no higher minor sbt version ?

Reply:
above https://github.com/leobenkel/Soteria/blob/cab958acf9724639429053d9cc572a83052d1797/project/build.properties#L1 ?

Reply:
seem to be `1.2.8` https://www.lightbend.com/blog/scala-sbt-128-patchnotes

Reply:
Yeah, but it doesn't fix that problem :(

Reply:
On top of that... when I moved to 1.2.8, for some reason it's no longer including commons-logging, so ParentTest.scala doesn't compile anymore.

Reply:
ok